packages/sampler/stories/qa/Scroller.js

```diff
@@ -158,11 +174,13 @@ storiesOf('Scroller', module)
 		'Horizontal scroll',
 		() => (
 			<Scroller
-				direction={select('direction', prop.direction, Scroller, 'horizontal')}
-				focusableScrollbar={boolean('focusableScrollbar', Scroller, false)}
-				horizontalScrollbar={select('horizontalScrollbar', prop.horizontalScrollbar, Scroller, 'auto')}
+				focusableScrollbar={boolean('focusableScrollbar', Config)}
```

Comment:
Please keep `direction` prop here.

Reply:
Restored. The title of this story is *Horizontal Scroller'  So, I removed the code line. Please check it.

Reply:
Doesn't really make sense to have direction prop here, but leaving it as-is.